tutorials/text/usetex.py

```diff
@@ -51,6 +51,32 @@
       "font.family": "serif",
       "font.serif": ["Palatino"],
   })
+  # It's also possible to use the reduced notation by directly setting font.family:
+  plt.rcParams.update({
+    "text.usetex": True,
+    "font.family": "Helvetica"
+  })
+
+Currently, the supported fonts are::
+
+    +---------------------+--------------------------------------------------+
```

Comment:
Looking at the rendered docs, I don't think the table should be indented; it appears entirely in code-style source form.